src/db/migrations/29-add-holly-plus.ts

```diff
@@ -0,0 +1,46 @@
+import { Knex } from 'knex';
+
+import { IdExtractorTypes, TitleExtractorTypes, WebsiteUrlExtractorTypes } from '../../types/fieldExtractor';
+import { NftFactory, NFTContractTypeName, NFTStandard } from '../../types/nft';
+import { MusicPlatform, MusicPlatformType } from '../../types/platform';
+import { addNftFactory, addPlatform, removeNftFactory, removePlatform } from '../migration-helpers';
+
+const HOLLY_PLUS_PLATFORM: MusicPlatform = {
+  id: 'hollyplus',
+  type: MusicPlatformType['multi-track-multiprint-contract'],
+  name: 'hollyplus',
+}
+
+const HOLLY_PLUS: NftFactory = {
+  id: '0x6688Ee4E6e17a9cF88A13Da833b011E64C2B4203',
+  startingBlock: '13001584',
+  platformId: 'hollyplus',
+  contractType: NFTContractTypeName.default,
+  standard: NFTStandard.ERC721,
+  autoApprove: true,
+  approved: true,
+  typeMetadata: {
+    overrides: {
+      artist: {
+        name: 'hollyplus',
+        websiteUrl: 'https://auction.holly.plus/',
+      },
+      type: MusicPlatformType['multi-track-multiprint-contract'],
+      extractor: {
+        title: TitleExtractorTypes.METADATA_NAME,
+        id: IdExtractorTypes.USE_TITLE_EXTRACTOR,
+        websiteUrl: WebsiteUrlExtractorTypes.USE_TOKEN_ID_APPENDED_EXTERNAL_URL,
```

Comment:
This is nice! Good example of doing the extra step/extra QA to get the best data